addons/l10n_lt/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,322 @@
+        <field name="account_id" ref="account_account_template_4492"/>
+        <field name="refund_account_id" ref="account_account_template_4492"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_tax_template_sales_0_vat13" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Sale 0% VAT (VAT13) goods for EU VAT payers</field>
+        <field name="description">0% VAT</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount" eval="0.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_4492"/>
+        <field name="refund_account_id" ref="account_account_template_4492"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_tax_template_sales_0_vat15" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Sale 0% VAT (VAT15) Service not in EU territory</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount" eval="0.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_4492"/>
+        <field name="refund_account_id" ref="account_account_template_4492"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <!-- 5% VAT -->
+    <record id="account_tax_template_sales_5" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Sale 5% VAT (VAT3)</field>
+        <field name="description">5% VAT</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount" eval="5.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_4492"/>
+        <field name="refund_account_id" ref="account_account_template_4492"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <!-- 21% VAT -->
+    <record id="account_tax_template_sales_21" model="account.tax.template">
+        <field name="tax_group_id" ref="tax_group_vat"/>
+        <field name="name">Sale 21% VAT (VAT1)</field>
+        <field name="description">21% VAT</field>
+        <field name="type_tax_use">sale</field>
+        <field name="amount" eval="21.0"/>
+        <field name="amount_type">percent</field>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="account_id" ref="account_account_template_4492"/>
+        <field name="refund_account_id" ref="account_account_template_4492"/>
+        <field name="sequence" eval="1"/>
+    </record>
+    <!-- Reversed Taxes -->
```

Comment:
Do you think it can still give you your desired results if you make Reverse Sale/Purchase taxes and Assumed Purchase group taxes, same as we have here https://github.com/odoo/odoo/blob/12.0/addons/l10n_be/data/account_tax_template_data.xml#L1061 for Belgium localization ? If yes, then you can also use group tax in fiscal position for `EU` instead of mapping two taxes there.

Reply:
Added group taxes for better usability of reversed/assumed taxes and now it is used in EU fiscal position.

Reply:
I am just a bit confused that the parent tax is called "Reversed" while it is within EU and it contains a normal and a reversed tax.

Reply:
Can you elaborate more what is confusing here?
What do you mean by within EU? And By parent reversed tax you mean `account_tax_template_sales_reversed_21`? It is used to display taxes, but not count it in accounting just as it is done on BE localization. It was actually redesigned following BE localization (previously we did not use grouped taxes).

Reply:
I mean it is intra-community.  The parent tax is called reverse while it contains e.g. two children that are +20% and -20%.  This -20% seems to be the reverse for me.  (and you call it that, but also the parent)  And the parent could be "intra-community" or something similar.

Reply:
Here in LT, for some reason we just use term reverse taxes, though I guess Intra Community is more correct term for such tax (that actually groups reverse tax)?

Reply:
I talked with our Finance Manager, so she said reverse taxes are also used inside Lithuania too (for construction business). So Intra-Community would indicate something between counties group like EU? And also she said everyone just use term reverse tax in Lithuania. So when selecting such tax manually on invoice for example, I guess it would be more clear for users the way it is named now?

We used group tax, just for user convenience, so he would not need to select two taxes (positive and negative), when it involves those reversed taxes.

Reply:
It just seemed weird to me, but if that is the clearest for you, let's keep it that way.